docs/how-to-guides/effective-gno.md

```diff
@@ -0,0 +1,652 @@
+---
+id: 'effective-gno'
+---
+
+# Effective Gno
+
+Welcome to the guide for writing effective Gno code. This document is designed
+to help you understand the nuances of Gno and how to use it effectively.
+
+Before we dive in, it's important to note that Gno shares several similarities
+with Go. Therefore, if you haven't already, we highly recommend reading
+["Effective Go"](https://go.dev/doc/effective_go) as a primer.
+
+## Disclaimer
+
+Gno is a young language. The practices we've identified are based on its current
+state. As Gno evolves, new practices will emerge and some current ones may
+become obsolete. We welcome your contributions and feedback. Stay updated and
+help shape Gno's future!
+
+## Counter-intuitive good practices
+
+This section highlights some Gno good practices that might seem
+counter-intuitive, especially if you're coming from a Go background.
+
+### Embrace global variables in realms
+
+In Gno, using global variables is not only acceptable, but it's also encouraged,
```

Comment:
Typo here, should be `global variables are...`, and `they're also...`

Reply:
Could someone please double-check? I believe the original source is accurate.

Reply:
@thehowl you're completely right, I missed the `using` completely 🤦‍♂️